en/firefox/developer.ftl

```diff
@@ -86,9 +86,21 @@ firefox-developer-view-the-release = View the <a { $attrs }>release notes</a> (E
 firefox-developer-developer-tools-user = Developer Tools User Docs
 firefox-developer-mdn-web-docs = { -brand-name-mdn-web-docs }
 firefox-developer-resources-for-developers = Resources for Developers, by Developers
+firefox-developer-mdn-playground = Playground
+firefox-developer-mdn-write-test-and-share = Write, test and share your code. Your playground to learn and share your amazing work with the world.
+firefox-developer-mdn-blog = MDN Blog
+firefox-developer-mdn-unlock-the-world = Unlock the world of web development with the MDN Blog - your go-to hub for expert insights, latest web standards, and coding tips.
+firefox-developer-mdn-updates = Updates
+firefox-developer-mdn-the-web-doesnt-have = The Web doesn’t have a changelog, but MDN can help. You can personalize and filter compatibility changes based on browsers or the tech category you are interested in whether that is JavaScript, CSS, etc.
```

Comment:
```suggestion
firefox-developer-mdn-the-web-doesnt-have = The web doesn’t have a changelog, but MDN can help. You can personalize and filter compatibility changes based on browsers or the tech category you are interested in whether that is JavaScript, CSS, etc.
```

I believe web is supposed to be lowercase?
https://acorn.firefox.com/latest/content/word-list/u-z-cTJ4bt1Z#section-web-e2